lib/sdf/link.rb

```diff
@@ -18,5 +18,51 @@ def each_sensor
         def pose
             Conversions.pose_to_eigen(xml.elements["pose"])
         end
+
+        # Check if link is kinematic
+        #
+        def kinematic?
+            if kinematic = xml.elements["kinematic"]
+                 Conversions.to_boolean(kinematic)
+            end
+        end
+
+        # The link's inertial.
+        #
+        # @return Inertial struct
+        Inertial = Struct.new(:pose, :mass, :inertia)
+        def inertial
+            mass = ixx = iyy = izz = 1
+            ixy = ixz = iyz = 0
+            pose = Eigen::Isometry3.new
+            if inertial = xml.elements["inertial"]
+                mass = read_float_child_element(inertial, "mass", 1)
+                pose = Conversions.pose_to_eigen(inertial.elements["pose"])
```

Comment:
It is not required. But the [Conversion.pose_to_eigen](https://github.com/rock-control/control-ruby_sdformat/blob/master/lib/sdf/conversions.rb#L26) returns a default value of <0 0 0 0 0 0>